dist.ini

```diff
@@ -14,10 +14,9 @@ format = %-8v %{yyyy-MM-dd}d
 [MetaJSON]
 
 [MetaResources]
```

Comment:
There's actually no need to set any of this explicitly. I just need to switch this to use my `@DROLSKY` bundle.